### ISSUE TITLE
Specifying dbname parameter during create causes an exception

### ISSUE THREAD

**Issue by lewisdawson** [db-migrate/node-db-migrate#294]
If you specify a `dbname` parameter during a create operation, as the documentation suggests you can, you get an exception.

> Usage: db-migrate [up|down|reset|create|db] __[[dbname/]migrationName|all]__ [options]

```bash
$ ../node_modules/db-migrate/bin/db-migrate create metadata/create-my-collection -e test
[ERROR] Error: ENOENT, open '<path>/db/migrations/metadata/20150819013642-metadata.js'
    at Error (native)
```

In addition, the name of the migration that's used is the `dbname` parameter, not the `migrationName` parameter.

**Comment by wzrdtales** [comment]
Ic, this information is indeed misleading.

I don't even know right now, why I have written like that. It is the plan to replace optimist soon and to rewrite the whole cli. I will correct the documentation, to make them fit what you actuall  can do. Thank you very much for reporting this issue!

For you a first version of the correction I'm going to apply to the master, please note that coffee script migrations of sql-file kind are not included within v0.9.x yet:


# Creating Migrations

DB-Migrate can be used to create migration boilerplates from templates. Currently there are only basic templates available, which may change in some future versions.

### Parameters
* migration name
* flags

### Flags
* coffee-file
* sql-file

The flags `--coffee-file` and `--sql-file` can be combined to create sql-file migrations in coffee script.

**Comment by wzrdtales** [comment]
Ok, I now know why it was noted like that.
Some time back, when I introduced scoped migrations, see https://github.com/db-migrate/node-db-migrate#multiple-migration-scopes for this topic, there has been a first version, which used parameters instead of the `up:scope` notation. There is no such parameter like "all" in any of the functionalities, but there was for the first version of scoping.

I'm going to correct this version also in the cli output of v0.9.x, thank you again for reporting this.
